src/JJMasterData.Core/Web/Html/HtmlBuilder.Attributes.cs

```diff
@@ -21,7 +21,8 @@ public HtmlBuilder WithNameAndId(string id)
         /// </summary>
         public HtmlBuilder WithAttribute(string name, string value)
         {
-            _attributes.Add(name, value);
+            if(!_attributes.ContainsKey(name))
```

Comment:
Why ignore the command? In my humble opinion, it should throw an exception or change the existing one.

Reply:
> Why ignore the command? In my humble opinion, it should throw an exception or change the existing one.

It's causing an exception in a existing system preventing a page from loading. But I think the intention of who is setting the attribute again is overwriting the existing value.